relayer/crypto/sr25519/sr25519.go

```diff
@@ -27,7 +27,7 @@ func GenerateKeypair(network uint8) (*Keypair, error) {
 	return NewKeypairFromSeed("//"+hexutil.Encode(data), network)
 }
 
-func NewKeypairFromSeed(seed string, network uint8) (*Keypair, error) {
+func NewKeypairFromSeed(seed string, network uint16) (*Keypair, error) {
```

Comment:
Ditto, seems unrelated.

Reply:
Removed in https://github.com/Snowfork/snowbridge/pull/981/commits/d1a5b8648f7f4b44041f3b9260667989a290cba2